lib/bodyScrollLock.esm.js

```diff
@@ -114,6 +115,7 @@ var setPositionFixed = function setPositionFixed() {
       document.body.style.position = 'fixed';
       document.body.style.top = -scrollY + 'px';
       document.body.style.left = -scrollX + 'px';
+      document.body.style.right = '0px';
```

Comment:
Is not needed set measure unit when value equals of 0

Reply:
Fixed